.github/workflows/tests.yaml

```
@@ -342,39 +342,6 @@ jobs:
         source: /tmp/artifacts
       if: always()
 
```

should be skipped instead of removed:

```suggestion
  utapi-v2-tests:
    if: false
```